Resources/Prototypes/DeltaV/Entities/Objects/Decoration/flora.yml

```diff
@@ -0,0 +1,100 @@
+- type: entity
+  id: BushBase
+  name: Bush
+  description: A pretty bush.
+  placement:
+    mode: PlaceFree
+  components:
+  - type: SpriteFade
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Sprite
+    noRot: true
+    sprite: DeltaV/Objects/Decoration/Flora/bush.rsi
+    state: base
+    drawdepth: Mobs
+  - type: Tag
+    tags:
+      - Structure
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Wood
+  - type: MeleeSound
+    soundGroups:
+      Brute:
+        path: /Audio/Effects/chop.ogg
+        params:
+          variation: 0.05
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 10
+      behaviors:
+        - !type:DoActsBehavior
+          acts: [ "Destruction" ]
+        - !type:PlaySoundBehavior
+          sound:
+            collection: WoodDestroy
+        - !type:SpawnEntitiesBehavior
+          spawn:
+            MaterialWoodPlank:
+              min: 0
+              max: 1
+
+- type: entity
+  parent: BushBase
+  id: BushLightBase
+  name: Light Bush
+  description: A pretty bush!
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Decoration/Flora/bush.rsi
```

Comment:
sprite path is inherited you only have to specify the layers

Reply:
👍